lxd/images.go

```diff
@@ -151,29 +252,29 @@ func extractTar(fname string) (int, error) {
 		}
 	*/
 
-	tr := tar.NewReader(f)
-	for {
-		hdr, err := tr.Next()
-		if err != nil {
-			if err == io.EOF {
-				break
-			}
-			fmt.Printf("got error %s\n", err)
-			/* TODO - figure out why we get error */
-			return 0, nil
-			//return 0, err
-		}
-		if hdr.Name != "metadata.yaml" {
-			continue
-		}
-		//tr.Read()
-		// find architecture line
-		break
-	}
+	//	tr := tar.NewReader(f)
+	//	for {
+	//		hdr, err := tr.Next()
+	//		if err != nil {
+	//			if err == io.EOF {
+	//				break
+	//			}
+	//			fmt.Printf("got error %s\n", err)
+	//			/* TODO - figure out why we get error */
+	//			return 0, nil
+	//			//return 0, err
+	//		}
+	//		if hdr.Name != "metadata.yaml" {
+	//			continue
+	//		}
+	//		//tr.Read()
+	//		// find architecture line
+	//		break
+	//	}
```

Comment:
Can we just axe this commented out code (and the stuff below)? I don't think there's any reason to keep it.